spec/formatting.md

```diff
@@ -175,7 +176,7 @@ whether its value was originally _quoted_ or _unquoted_.
 > For example,
 > the _option_ `foo=42` and the _option_ `foo=|42|` are treated as identical.
 
-The resolution of a _text_ or _literal_ token MUST always succeed.
+The resolution of a _text_ or _literal_ token MUST never fail to resolve.
```

Comment:
"MUST never" is not proper 2119

```
The resolution of a _text_ or _literal_ token MUST NOT fail to resolve.
```

... which is still a double negative, so the "suggestion" would be:

```suggestion
The resolution of a _text_ or _literal_ MUST resolve to a string.
```